src/SecureStream.php

```diff
@@ -4,12 +4,11 @@
 
 use Evenement\EventEmitterTrait;
 use React\EventLoop\LoopInterface;
-use React\Stream\DuplexStreamInterface;
 use React\Stream\WritableStreamInterface;
 use React\Stream\Stream;
 use React\Stream\Util;
 
-class SecureStream extends Stream implements DuplexStreamInterface
+class SecureStream extends Stream
```

Comment:
Wasn't aware `DuplexStreamInterface` was added later in 0.4.x, so yeah it isn't a BC break imo :).